subtests/docker_cli/run_cidfile/run_cidfile.py

```diff
@@ -137,6 +137,8 @@ def run_once(self):
                                                lambda x: mustfail(x, 125)))
         self._check_failure_cidfile_present(containers[-1])
         # cidfile already exists (exited container)
+        # FIXME: this occasionally throws SIGPIPE, presumably because
+        #  container isn't fully ready. This is a tough one to solve.
```

Comment:
Ahh right, okay, I'll keep an eye out for them.  Hopefully your other race-fixes will help with this test as well.